### PR TITLE
Handle redirects during tabbed-view requests.

### PR DESCRIPTION
**Pull request by deiferni**
When a users session is removed (timeout, logout in another tab, ...) or when the response is redirected for another reason tabbed-view layout can be broken when attempting to render the response:
before

This PR addresses that issue by adding a custom header on the server side to each tabbed-view response. The client then checks if the header is present, if this is not the case the response is discarded and error message is displayed:
after

I did not want to trigger a reload all the time to avoid reload-loops when the the page containing the tabbed-view can be loaded, but only the tab fails/redirects for some reason. If a custom action is required this can be implemented by adding an event handler, e.g.:

```javascript
$(document).delegate('body', 'tabbedview.unknownresponse', function(event, overview, jqXHR) {
  if (jqXHR.getResponseHeader('X-GEVER-Service') === 'Portal') {
    // the response we got originates from a portal, we reload to show the login window
    location.reload();
    return false;
  }
});
```

@jone or @maethu please have a good look (if you're interested a demo of the implementation is currently running on https://lab.onegovgever.ch/)

Note that this will break custom `TabbedView` implementations, that do not set the required header in `listing`, should they exist.

This PR also adds a development configuration to be able to use the `i18n-build` script. It also removes i18n_domain attribute from config files that are not translated.

### CODE REVIEW

**Comment by jone** [comment]
Did you think about ExtJS requests with JSON responses? E.g. when sorting a listing table.

File: ftw/tabbedview/browser/tabbed.py [hunk 1]
```diff
@@ -284,3 +285,15 @@ def set_default_tab(self, tab=None, view=None):
         storage.set(key, tab.lower())
 
         return json.dumps(success)
+
+    def msg_unknownresponse(self):
+        """Return the message that is rendered when a javascript request gets
+        a response from a differnt source than a tabbed view.
```
Comment (jone) [inline ftw/tabbedview/browser/tabbed.py]: typo `differnt`

**Comment by jone** [comment]
I think all tabs should go through the `tabbed_view/listing` method when being loaded, thus all tabs should have the header. I don't see any problem with custom tabs.

Just ExtJS is probably not covered, it might be out of scope.

:+1: I think this is fine

**Comment by deiferni** [comment]
@jone updated: typo fixed.

As discussed ExtJS is out of scope since hooking into ExtJS request handling does not seem to be that simple.

**Comment by lukasgraf** [comment]
Looks good to me! :+1: :cake:

**Pull request merged**